motioneye/static/js/main.js

```diff
@@ -4973,13 +4980,15 @@ function addCameraFrameUi(cameraConfig) {
         })
     });
 
-    if (cameraConfig.actions.length <= 4) {
-        cameraOverlay.find('div.camera-overlay-bottom').addClass('few-buttons');
-    }
-    else {
-        //cameraOverlay.find('div.camera-action-buttons-wrapper').css('width', Math.ceil(cameraConfig.actions.length / 2) * 2.5 + 'em');
-        cameraOverlay.find('div.camera-action-buttons-wrapper').css('width', 4 * 2.5 + 'em');
-    }
+    var actionsCount = Math.ceil(cameraConfig.actions.length);
+
+    // if (cameraConfig.actions.length <= 4) {
+    //     cameraOverlay.find('div.camera-overlay-bottom').addClass('few-buttons');
+    // }
+    // else {
+    //     cameraOverlay.find('div.camera-action-buttons-wrapper').css('width', Math.ceil(cameraConfig.actions.length / 2) * 2.5 + 'em');
+    //     //cameraOverlay.find('div.camera-action-buttons-wrapper').css('width', 4 * 2.5 + 'em');
+    // }
 
```

Comment:
Let's do not keep old code commented. No one will ever be able to understand how this was used, with the CSS changed. And if someone wants to know how it worked before, on Git(Hub) the PR/commits can be checked, or an older commit checked out:
```suggestion
```

Reply:
Yes off course. Sry about that. Done